source/ProtonPack/ProtonPack.ino

```diff
@@ -1121,6 +1231,114 @@ void powercellOff() {
   i_powercell_led = 0;
 }
 
+void powercellReset() {
+  for(int i = 0; i <= i_powercell_led; i++) {
+    uint8_t r = 0;
+    uint8_t g = 0;
+    uint8_t b = 0 + (255 * i_powercell_brightness / 100);
```

Comment:
This calculation appears quite a bit now. Recommend making a helper function to do this adjustment. Conversely, if the value might be calculated when the dimming adjustment is made, that value could be a global and simply used in each location where this calculation is performed (it's minor, but every math operation adds up given the number of times the LED's transition).

Reply:
Cleaned it up a bit now. Only appearing once now.